src/app/core/services/repo-creator.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@angular/core';
+import { GithubService } from './github.service';
+import { UserService } from './user.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class RepoCreatorService {
```

Comment:
Let's remove `RepoCreatorService` from this PR and introduce it when we create the subsequent custom operators.

Reply:
👍  Alright